### DISCUSSION TITLE
Saving Arabic letters in some forms shouldn't have a blank margin

### DISCUSSION THREAD

**Discussion by Youssef1313** [character-map-uwp/Character-Map-UWP#295] [Ideas]
For example, `\uFED4` (Unicode Character 'ARABIC LETTER FEH MEDIAL FORM' (U+FED4)) in Times New Roman font is saved as:

![ـفـ](https://user-images.githubusercontent.com/31348972/209205777-9c7b30c6-dfa3-4c82-ac7b-4f0f354de41b.png)

The blank margin in both sides shouldn't be there, I think.

Another example, `\uFED2` (Unicode Character 'ARABIC LETTER FEH FINAL FORM' (U+FED2)) in Times New Roman font is saved as:

![ـف](https://user-images.githubusercontent.com/31348972/209206066-8a0de9e1-4c43-4def-91e9-56897f9268ad.png)

The margin at the right is probably correct (not too sure), but on the left there shouldn't be a margin.

**Comment by JohnnyWestlake** [comment]
Current logic for saving characters is to always save them centered by glyph draw bounds inside a box, it doesn't actually save to glyph bounds themselves (to ensure a consistent size), so at the moment this is "expected". If you want each glyph export clipped to draw bounds (like SVG export does) that's another option that could be considered.
